src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -57,38 +58,53 @@ public Command parseCommand(String userInput) throws ParseException {
         switch (commandWord) {
 
         case AddCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", AddCommand.COMMAND_WORD));
             return new AddCommandParser().parse(arguments);
 
         case EditCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", EditCommand.COMMAND_WORD));
             return new EditCommandParser().parse(arguments);
 
         case DeleteCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", DeleteCommand.COMMAND_WORD));
             return new DeleteCommandParser().parse(arguments);
 
         case ClearCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", ClearCommand.COMMAND_WORD));
             return new ClearCommand();
 
         case FindCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", FindCommand.COMMAND_WORD));
             return new FindCommandParser().parse(arguments);
 
         case ListCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", ListCommand.COMMAND_WORD));
             return new ListCommand();
 
         case ExitCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", ExitCommand.COMMAND_WORD));
             return new ExitCommand();
 
         case HelpCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", HelpCommand.COMMAND_WORD));
             return new HelpCommand();
 
         case AddNotesCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", AddNotesCommand.COMMAND_WORD));
             return new AddNotesCommandParser().parse(arguments);
 
         case ViewCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", ViewCommand.COMMAND_WORD));
             return new ViewCommandParser().parse(arguments);
 
         case AddAppointmentCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", AddAppointmentCommand.COMMAND_WORD));
             return new AddAppointmentCommandParser().parse(arguments);
 
+        case DeleteAppointmentCommand.COMMAND_WORD:
+            logger.info(String.format("COMMAND WORD: \"%s\"", DeleteAppointmentCommand.COMMAND_WORD));
```

Comment:
i think instead of repeated lines of logger.info, could consolidate it into a single line outside of the block, since its going to take the commandWord value as an argument anyways

Reply:
good catch! will amend accordingly

Reply:
Good point, although my intention was actually to show the path taken through this switch case, to verify the correct path taken on the correct command. Maybe it was not very clear, will amend the logged message!